### PR TITLE
Initial create and update SQL code editor docs

### PR DESCRIPTION
**Pull request by alexanderahn**
https://app.asana.com/0/1203457878958265/1207584391856672

### CODE REVIEW

File: docs/SQL/development/visual-editor/interactive-development/interactive-development.md [hunk 1]
```diff
@@ -12,6 +12,14 @@ tags:
   - data
 ---
 
+You can leverage Data Copilot for SQL by building your Model and adding individual Gems, and then check how the process is going by interactively running the model. Other interactive development features include:
+
+- Data Explorer
+- Interim
```
Comment (atbida) [inline docs/SQL/development/visual-editor/interactive-development/interactive-development.md:18]: The "Data Explorer" feature is essentially the umbrella term for the features: interim, sort, filter, download. I'd suggest:

You can leverage Data Copilot for SQL by building your Model and adding individual Gems, and then check how the process is going by interactively running the model. Use the Data Explorer to dig into your interim data sample, with the ability to sort, filter, and download your sample data.

Comment (alexanderahn) [reply docs/SQL/development/visual-editor/interactive-development/interactive-development.md:18]: Got it.

File: docs/SQL/development/development.md [hunk 2]
```diff
@@ -56,12 +42,39 @@ Vast majority of dbt and SQL constructs are supported within the Prophecy Visual
...
+```mdx-code-block
+import DocCardList from '@theme/DocCardList';
+import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
 
-2. [**Gems**](./gems/gems.md) - The individual steps, or transformations, to define a table or view.
+<DocCardList items={useCurrentSidebarCategory().items}/>
+```
 
-As you're building your Model by adding individual Gems, see how the process is going by interactively running the model. Explore the [Interactive Development](./interactive-development/data-explorer.md) page to learn more.
+As you're building your Model by adding individual Gems, see how the process is going by interactively running the model. To learn more, explore the [Data Explorer](visual-editor/interactive-development/data-explorer) page under Interactive Development.
```
Comment (atbida) [inline docs/SQL/development/development.md:80]: Nit: "individual Gems" can link to /SQL/gems/

File: docs/Spark/Spark.md [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
+tags: [spark, warehouse]
 ---
 
+Prophecy Data Transformation Copilot for Spark combines the best of both worlds; **high-quality code** based on software engineering best practices with a complete, **easy-to-use visual** environment.
+
+![CodeToVisual](./img/CodeToVisualSpark.png)
+
 ```mdx-code-block
 import DocCardList from '@theme/DocCardList';
 import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
```
Comment (atbida) [inline docs/Spark/Spark.md:14]: When I click on the links in the "Standard Gems" list here, the page is not found: 
/Spark/gems/

Not sure if that's caused by this PR or a previous one.

**Review by atbida — COMMENTED** [review]
this looks great to me, we just need to confirm the redirects are working.

File: docs/SQL/development/development.md (hunk 2 above)
Comment (alexanderahn) [reply docs/SQL/development/development.md:80]: Done.

File: docs/Spark/Spark.md (hunk 3 above)
Comment (atbida) [reply docs/Spark/Spark.md:14]: This might get fixed when we re-index the corpus after merging the PR.

**Review by atbida — APPROVED** [review]
lgtm; we can reindex and verify the links on the /spark/gems/* pages work

File: docs/Spark/Spark.md (hunk 3 above)
Comment (alexanderahn) [reply docs/Spark/Spark.md:14]: Okay, I'll check and re-index if needed.

**Review by atbida — APPROVED** [review]
lgtm!

**Pull request merged**